config/planner.yaml

```diff
@@ -0,0 +1,56 @@
+controller_frequency: 5.0
+recovery_behaviour_enabled: true
+
+NavfnROS:
+  allow_unknown: true # Specifies whether or not to allow navfn to create plans that traverse unknown space.
+  default_tolerance: 0.1 # A tolerance on the goal point for the planner.
+
+TrajectoryPlannerROS:
+  # Robot Configuration Parameters
+  # Acceleration parameters
+  acc_lim_x: 0.5
+  acc_lim_theta: 1.0
+
+  max_vel_x: 0.3
+  min_vel_x: 0.2
+  max_vel_y: 0.3
+  min_vel_y: 0.2
+  max_vel_theta: 1.0
+  min_vel_theta: -1.0
+
+  min_in_place_vel_theta: 1.0
+
+  holonomic_robot: true
+  escape_vel: -0.1
+
+  # Goal Tolerance Parameters
+  yaw_goal_tolerance: 1.0
+  xy_goal_tolerance: 1.0
```

Comment:
1 radian and 1 meter for angular and translational tolerances are waaaay too huge. I guess these numbers were chosen deliberately off for workshops and here should be config that runs out-of-the-box without modification.

Reply:
i agree with veiko, we should put smaller values here. but still something that works reliably. if the tolerances are too small we get oscillations and failures - which we don't want :)
so @mmaao if you can figure out smaller tolerances that in testing give stable results, please change the values here.